examples/withdraw_from_derive.py

```diff
@@ -0,0 +1,221 @@
+"""
+Example of bridging funds from a Derive smart contract funding account to BASE
+"""
+
+import json
+import os
+
+from dotenv import load_dotenv
+from eth_account import Account
+from web3 import Web3
+from web3.contract import Contract
+
+from derive_client.bridge.constants import MSG_GAS_LIMIT, PAYLOAD_SIZE, TARGET_SPEED
+from derive_client.bridge.enums import ChainID, Currency, DRPCEndPoints, TxStatus
+from derive_client.bridge.models import Address
+from derive_client.bridge.transaction import sign_and_send_tx
+from derive_client.bridge.utils import (
+    estimate_fees,
+    exp_backoff_retry,
+    get_contract,
+    get_erc20_contract,
+    get_prod_lyra_addresses,
+    get_repo_root,
+)
+from derive_client.derive import DeriveClient
+from tests.conftest import Environment
+
+CONTROLLER_ABI_PATH = get_repo_root() / "data" / "controller.json"
+DEPOSIT_HOOK_ABI_PATH = get_repo_root() / "data" / "deposit_hook.json"
+LIGHT_ACCOUNT_ABI_PATH = get_repo_root() / "data" / "light_account.json"
+L1_CHUG_SPLASH_PROXY_ABI_PATH = get_repo_root() / "data" / "l1_chug_splash_proxy.json"
+L1_STANDARD_BRIDGE_ABI_PATH = get_repo_root() / "data" / "l1_standard_bridge.json"
+WITHDRAW_WRAPPER_V2 = get_repo_root() / "data" / "withdraw_wrapper_v2.json"
+
+
+def check_allowance(
+    token_contract: Contract,
+    owner: Address,
+    spender: Address,
+):
+    allowance = token_contract.functions.allowance(owner, spender).call()
+    return allowance
+
+
+def bridge_mainnet_eth_to_derive(
+    account: Account,
+    amount: int,
+):
+    w3_mainnet = Web3(Web3.HTTPProvider(DRPCEndPoints.ETH))
+    if not w3_mainnet.is_connected():
+        raise ConnectionError("Failed to connect to Ethereum mainnet RPC.")
+
+    proxy_address = "0x61e44dc0dae6888b5a301887732217d5725b0bff"
+    bridge_abi = json.loads(L1_STANDARD_BRIDGE_ABI_PATH.read_text())
+    proxy_contract = get_contract(w3=w3_mainnet, address=proxy_address, abi=bridge_abi)
+
+    eth_balance = w3_mainnet.eth.get_balance(account.address)
+    nonce = w3_mainnet.eth.get_transaction_count(account.address)
+
+    @exp_backoff_retry
+    def simulate_tx():
```

Comment:
take w3 and txn such that cna simulate txns. 
Potentially then helpful as can be bundled with a txn and then simulated till txns then signed and submitted.